backend/src/Designer/Models/FooterFile.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Altinn.Studio.Designer.Models;
+
+public class FooterFile
+{
+    [JsonPropertyName("$schema")]
+    public string Schema { get; set; }
+
+    [JsonPropertyName("footer")]
+    public List<Footer> Footer { get; set; }
+}
+
+public class Footer
```

Comment:
A couple of comments:
1. In the [JSONSchema](https://github.com/Altinn/altinn-cdn/blob/master/toolkits/altinn-app-frontend/4/schemas/json/layout/footer.schema.v1.json) for this file, the definition is called `FooterLayout`, I suggest we use the same for this class name.
2. If you look at the JSONSchema definition, you can see that the properties here are dependent on which type of component is used. Only the `Link` type component actually supports all these properties. We should consider typing the component types here as well, or at the very least making the optional properties nullable to indicate that they are not in fact required.

Reply:
Smart! Pushet a more specific definition now, but not 100% sure ut is correct 🫣